### PR TITLE
Removed examples and example files

### LINKED ISSUE #51: Remove example models and endpoints
#### Description
When I set up the backend, I put models for Emails and Persons with endpoints as examples so that we could ensure that everyone's connections to the DB were working. Delete those now!

#### Files
- backend/api/models/Email.py
- backend/api/models/Person.py
- backend/api/views/main.py
- backend/api/models/__init__.py

#### Tasks
Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.
- [x] Delete models
- [x] Delete endpoints
- [x] Search for them in the repo to make sure that you've deleted everything you need to
- [x] Make sure you didn't break the backend

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

Fixes #51 
Super quick removal 😄 
Lowkey, I feel like after our MVP we should implement unit testing for our endpoints and frontend too.

### CODE REVIEW

**Review by mc578 — APPROVED** [review]
thank you

**Review by Feeshay — APPROVED** [review]
thank you
also I hate using github with 2 accounts :/

File: backend/api/views/auth.py [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-
-
-@auth.route("/verifyEmail", methods=["POST"])
-def verify_email():
-    headers = {"Content-Type": "application/json", "token": request.headers.token}
-    data = {"pin": request.body.pin}
-    results = requests.post(
-        "http://localhost:8000/verifyEmail", headers=headers, data=data
-    )
-    return create_response(data={results.json()})
```
Comment (chyku) [inline backend/api/views/auth.py:16]: don't delete this this is for the auth server

File: backend/api/views/auth.py [hunk 2]
```diff
@@ -1,16 +1,6 @@
 from flask import Blueprint, request, jsonify
-from api.models import db, Person, Email
+from api.models import db
```
Comment (chyku) [inline backend/api/views/auth.py:2]: this is fine to keep though

**Review by chyku — CHANGES_REQUESTED** [review]
tiny thing

File: backend/api/models/User.py [hunk 3]
```diff
@@ -1,15 +0,0 @@
-from api.core import Mixin
-from .base import db
-from api.models import Email
-from flask_mongoengine import Document
```
Comment (chyku) [inline backend/api/models/User.py:4]: ALSO don't delete this mb user is also auth stuff

**Review by chyku — CHANGES_REQUESTED** [review]
another thing i missed

but wrt unit testing, maybe i have too much of a move fast mindset but i don't see much value in backend unit testing for a relatively straightforward small scale CRUD app. we're also pretty limited on time and i'd rather have us develop more features (or to finish what we want to in the first place!) than to move backwards and test get/delete endpoints when in reality they won't fail all that easily. 

i also am not a fan of frontend testing for small apps because unless several teams are using the same components and then someone changes that component & a jest snapshot fails (basically jest just takes "snapshots" of components in the current instance and if it changes, the snapshot test fails and you have to manually update it yourself) they can check to see if they broke anything.

**Review by chyku — APPROVED** [review]
:+1:

**Pull request merged**